documentation/config_file_puppetserver.markdown

```diff
@@ -73,6 +73,19 @@ The `puppetserver.conf` file contains settings for Puppet Server software. For a
 * The `profiler` settings configure profiling:
 
     * `enabled`: If this is set to `true`, Puppet Server enables profiling for the Puppet Ruby code. The default is `false`.
+    
+* The `product` settings configure update checking and analytics data collection:
```

Comment:
It seems like the headache that users will encounter from upgrades if the `product` key exists in puppetserver.conf would be pretty frustrating. I'm 👍 to having users place this setting in its own file `product.conf` within puppetservers config directory.